go.mod

```diff
@@ -1,6 +1,6 @@
 module k8s.io/cloud-provider-gcp
 
-go 1.15
```

Comment:
/cc @mikedanese Make sure https://golang.org/doc/go1.16#crypto/x509 does not provide any issues to any of the CSR controllers.

Reply:
The only thing significant is ParseCertificate change, which gets called on CSRs in kube-apiserver before gcp-controller-manager will ever see it. This looks fine.